src/components/util/checkbox-list.js

```diff
@@ -24,7 +23,7 @@ const CheckboxList = ({
         const newChecked = new Set(
             [...checked].filter((element) => {
                 for (const existingValue of existingValues) {
```

Comment:
You could replace the block with the `for` loop with `existingValues.some((existingValue) => existingValue.uuid === element.uuid)`